analyzers/rspec/vbnet/S3776.json

```diff
@@ -9,8 +9,10 @@
   },
   "status": "ready",
   "remediation": {
-    "func": "Constant\/Issue",
-    "constantCost": "10min"
+    "func": "Linear with offset",
```

Comment:
IMO this change should be explicit in the release notes, because it will affect the ratings